--- libmicrohttpd-0.9.28/src/microhttpd/daemon.c.orig	2013-07-19 17:32:13.000000000 +0200
+++ libmicrohttpd-0.9.28/src/microhttpd/daemon.c	2013-07-19 20:31:23.474232117 +0200
@@ -612,9 +612,11 @@
 	  break;
 	}
     }
+#if HAVE_MESSAGES
 #if DEBUG_CONNECT
   MHD_DLOG (daemon, "Maximum socket in select set: %d\n", *max_fd);
 #endif
+#endif
   return MHD_YES;
 }
 
@@ -3112,9 +3114,11 @@
 	  int sk_flags = fcntl (socket_fd, F_GETFL);
 	  if (0 != fcntl (socket_fd, F_SETFL, sk_flags | O_NONBLOCK))
 	    {
+#if HAVE_MESSAGES
 	      MHD_DLOG (daemon,
 			"Failed to make listen socket non-blocking: %s\n", 
 			STRERROR (errno));
+#endif
 	      if (0 != CLOSE (socket_fd))
 		MHD_PANIC ("close failed\n");	      
 	      goto free_and_fail;	      
