--- libmicrohttpd-0.9.29/src/microhttpd/daemon.c.orig	2013-08-22 08:09:22.000000000 +0200
+++ libmicrohttpd-0.9.29/src/microhttpd/daemon.c	2013-08-26 20:01:42.635866349 +0200
@@ -620,9 +620,11 @@
 	  break;
 	}
     }
+#if HAVE_MESSAGES
 #if DEBUG_CONNECT
   MHD_DLOG (daemon, "Maximum socket in select set: %d\n", *max_fd);
 #endif
+#endif
   return MHD_YES;
 }
 
@@ -1272,8 +1274,11 @@
     if ( (MHD_YES == external_add) &&
 	 (-1 != daemon->wpipe[1]) &&
 	 (1 != WRITE (daemon->wpipe[1], "n", 1)) )
+#if HAVE_MESSAGES
       MHD_DLOG (daemon,
-		"failed to signal new connection via pipe");
+		"failed to signal new connection via pipe")
+#endif
+	      ;
 
 #if EPOLL_SUPPORT
   if (0 != (daemon->options & MHD_USE_EPOLL_LINUX_ONLY))
